Configurations/15-android.conf

```diff
@@ -76,10 +76,13 @@
                 if (which("clang") !~ m|^$ndk/.*/prebuilt/([^/]+)/|) {
                     die "no NDK clang on \$PATH";
                 }
+                my $host=$1;
                 # harmonize with gcc default
-                (my $tridefault = $triarch) =~ s|^arm-|armv5te-|;
-                $cflags .= " -target $tridefault -gcc-toolchain "
-                        . "\$(ANDROID_NDK)/toolchains/$triarch-4.9/prebuilt/$1";
+                (my $tridefault = $triarch) =~ s/^arm-/armv5te-/;
+                (my $tritools   = $triarch) =~ s/(?:x|i6)86(_64)?-.*/x86$1/;
```

Comment:
So ...  `i686_64` is acceptable these days?  ;-)

But let me see if I get this right, for the x86 stuff, `$tritools` becomes 'x86' or 'x86_64', which for all others (say, 'arm-linux-androideabi'), `$tritools` becomes exactly equal to `$triarch` (so, 'arm-linux-androideabi'), is that right?

Reply:
Yes, that's how it looks in toolchains. Yes, it's inconsistent. Yes, you can sense even more inconsistency, e.g. .../toolchains/x86-4.9/prebuilt/linux-x86_64/i686-linux-android/ and .../toolchains/x86_64-4.9/prebuilt/linux-x86_64/x86_64-linux-android/ are real directories. No, I don't know what they were smoking...

Reply:
Okie.